imgui.h

```diff
@@ -1953,7 +1954,7 @@ struct ImGuiIO
 
     // Optional: Notify OS Input Method Editor of the screen position of your cursor for text input position (e.g. when using Japanese/Chinese IME on Windows)
     // (default to use native imm32 api on Windows)
-    void        (*SetPlatformImeDataFn)(ImGuiViewport* viewport, ImGuiPlatformImeData* data);
+    std::function<void(ImGuiViewport* viewport, ImGuiPlatformImeData* data)> SetPlatformImeDataFn;
```

Comment:
`std::function` is not necessary here and would be problematic for people using Dear ImGui from other languages. Your `SetPlatformImeDataFn` handler should not capture `bd`, it should use `ImGui_ImplOSX_GetBackendData` instead like all of the other backends.

(No comment on the rest of the PR, this just stood out to me while skimming.)

Reply:
OK.Thanks.I have modified the code.